etl/churn.kp/knowledge.md

```diff
@@ -496,7 +486,7 @@ def compute_churn_week(df, week_start, bucket, prefix):
                 x.get('start_version', 'unknown'),
                 x.get('sync_usage', 'unknown'),
                 x.get('current_version', 'unknown'),
-                x.get('current_week', 'unknown'),
+                x.get('current_week', -1),
```

Comment:
I like the idea of removing the 'unknown' values, I think it should be saved for another PR because it would require modifying both churn and churn_to_csv in tandem, and checking the output of churn_to_csv to make sure outputs are the same.

On the other hand, I think -1 is a good value for a default since `current_week` is an attribute that is grouped by. It has default values of -1 or null in `get_week_num`.